help/develop/function_results_cache.md

```diff
@@ -0,0 +1,83 @@
+# Caching function results
+
+Datagrok can cache the results of functions on both client and server sides.
```

Comment:
Also you have to expand here on what functions are (queries/computations/et), why caching might be a good idea (to improve responsiveness, reduce traffic, etc), and what are good candidates for caching (immutable, long-running functions producing not huge results that people execute over and over with the same parameters).